src/main/java/uk/gov/hmcts/reform/sscs/service/VenueService.java

```diff
@@ -24,6 +26,16 @@ public Optional<String> getEpimsIdForVenue(String processingVenue) {
             .map(VenueDetails::getEpimsId);
     }
 
+    public List<String> getActiveRegionalEpimsIdsForRpc(String rpc) {
+
+        return venueDataLoader.getActiveVenueEpimsIdsMapByRpc()
+                .entrySet()
+                .stream()
+                .filter(e -> rpc.equals(e.getKey()))
+                .map(Map.Entry::getValue)
+                .collect(Collectors.toList()).stream().collect(ArrayList::new, ArrayList::addAll, ArrayList::addAll);
```

Comment:
Is the final collect needed?

Reply:
As far as I understand yes, it's a nested List<List<String>> object, I did have a bit of a play with flatMap but ended on this as least cumbersome solution.